notebooks/llm-question-answering/llm-question-answering.ipynb

```diff
@@ -9,14 +9,14 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.            openvino_output = self.compiled_detokenizer([[0, token_id]])
</pre><p>It looks like some postprocessing logic that may be dependent from prompt format and specific model. Do you test it with different models?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2062/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
yes, for these models that works, in future it is looks like it could be replaced with tokenizer.decode()